pf9-autodeploy.yml

```diff
@@ -11,8 +11,20 @@
     - name: install python2
       raw: if [ -e /etc/lsb-release -a ! -e /usr/bin/python ]; then (apt -y update && apt install -y python-minimal); fi
 
-# Run pre_hook (if exists)
-- hosts: all
+# Run pre_flight_checks
+- hosts:
+    - hypervisors
+    - k8s-master
+    - k8s-worker
+  become: true
+  roles:
+    - pre-flight-checks
+
+# Run pre_hook
+- hosts:
```

Comment:
shouldn't this run on all? I'd anticipate most folks have pre/post checks on every node, not only {hyper,container}visors.